governance/program/src/state/enums.rs

```diff
@@ -56,6 +56,10 @@ pub enum GovernanceAccountType {
 
     /// Program metadata account. It stores information about the particular SPL-Governance program instance
     ProgramMetadata,
+
+    /// NativeTreasury account
+    /// The account has no data and can be used as payer for instructions signed by Governance PDAs or as native SOL treasury
+    NativeTreasury,
```

Comment:
Since we never deserialise the account the address derivation provides the required safety.
This account is also not even owned by `spl-governance` so it can't write anything to it, and then PDA with non zero data can't spend its lamports either.
I removed the enum value to avoid confusions